documentation/modules/vmware-prerequisites.adoc

```diff
@@ -5,27 +5,34 @@
 [id="vmware-prerequisites_{context}"]
 = VMware prerequisites
 
-It is strongly recommended to create a VDDK image to accelerate migrations. For more information, see xref:../master.adoc#creating-vddk-image_mtv[Creating a VDDK image].
-
 The following prerequisites apply to VMware migrations:
 
-* You must use a xref:compatibility-guidelines_{context}[compatible version] of VMware vSphere.
+* You must use a xref:compatibility-guidelines_mtv[compatible version] of VMware vSphere.
 * You must be logged in as a user with at least the minimal set of xref:vmware-privileges_{context}[VMware privileges].
 * To access the virtual machine using a pre-migration hook, link:https://www.vmware.com/support/ws5/doc/new_guest_tools_ws.html[VMware Tools] must be installed on the source virtual machine.
 * The VM operating system must be certified and supported for use as a link:https://access.redhat.com/articles/973163#ocpvirt[guest operating system with {virt}] _and_ for link:https://access.redhat.com/articles/1351473[conversion to KVM with `virt-v2v`].
 * If you are running a warm migration, you must enable link:https://kb.vmware.com/s/article/1020128[changed block tracking (CBT)] on the VMs and on the VM disks.
 * If you are migrating more than 10 VMs from an ESXi host in the same migration plan, you must increase the NFC service memory of the host.
-* It is strongly recommended to disable hibernation because {project-first} does not support migrating hibernated VMs.
 
 
 [IMPORTANT]
 ====
-In the event of a power outage, data might be lost for a VM with disabled hibernation. However, if hibernation is not disabled, migration will fail
+In the event of a power outage, data might be lost for a VM with disabled hibernation. However, if hibernation is not disabled, migration will fail.
+====
+
+[NOTE]
+====
+Neither {project-short} nor {virt} support conversion of Btrfs for migrating VMs from VMWare.
 ====
 
 [NOTE]
 ====
-Neither {project-short} nor OpenShift Virtualization support conversion of Btrfs for migrating VMs from VMWare.
+Creating a VDDK image, although optional, is highly recommended. Using {project-short} without VDDK is not recommended and could result in significantly lower migration speeds. If you are not using VDDK and a plan fails, then retry with VDDK installed. For more information, see xref:../master.adoc#creating-vddk-image_vmware[Creating a VDDK image].
```

Comment:
Pleas use the more strong wording `It is strongly recommended…` that we are also using above.